haskell-src/Concordium/Types/ProtocolLevelTokens/CBOR.hs

```diff
@@ -396,12 +396,12 @@ decodeCborTokenHolderHelper = \case
             Right coinInfo -> Just coinInfo
 
 -- | Decode a CBOR-encoded 'TokenHolder'.
```

Comment:
```suggestion
-- | Decode a CBOR-encoded 'CborHolderAccount'.
```

Reply:
I agree, also aligns better with the CIS7 description